Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorHelpers.cpp

```diff
@@ -63,23 +63,22 @@ namespace AzToolsFramework
     static const float s_iconFarDist = 40.f; // distance at which icons are at minimum scale
 
     // helper function to wrap EBus call to check if helpers are being displayed
-    // note: the ['?'] icon in the top right of the editor
     static bool HelpersVisible()
     {
         bool helpersVisible = false;
-        EditorRequestBus::BroadcastResult(helpersVisible, &EditorRequests::DisplayHelpersVisible);
+        EditorRequestBus::BroadcastResult(helpersVisible, &EditorRequestBus::Events::DisplayHelpersVisible);
         return helpersVisible;
     }
 
-    // calculate the icon scale based on how far away it is (distanceSq) from a given point
+    // calculate the icon scale based on how far away it is from a given point
     // note: this is mostly likely distance from the camera
-    static float GetIconScale(const float distSq)
+    static float GetIconScale(const float distanceFromCamera)
     {
         AZ_PROFILE_FUNCTION(AzToolsFramework);
 
         return s_iconMinScale +
             (s_iconMaxScale - s_iconMinScale) *
-            (1.0f - AZ::GetClamp(AZ::GetMax(0.0f, sqrtf(distSq) - s_iconCloseDist) / s_iconFarDist, 0.0f, 1.0f));
+            (1.0f - AZ::GetClamp(AZ::GetMax(0.0f, distanceFromCamera - s_iconCloseDist) / s_iconFarDist, 0.0f, 1.0f));
```

Comment:
I think for proper scaling this should be
(distance - close)/(far - close)

Reply:
Ah okay nice I can fix this 👍